src/frontend/src/handler/mod.rs

```diff
@@ -463,6 +463,9 @@ pub async fn handle(
             }
             DescribeKind::Plain => describe::handle_describe(handler_args, name),
         },
+        Statement::DescribeFragment { fragment_id } => {
```

Comment:
used a new syntax here because fragment_id is u32 and its semantically different from object_name from describe_fragments. Another approach would be adding a new `DescribeKind ` and serialize fragment id into object_name.